uber/utils.py

```diff
@@ -477,6 +477,31 @@ def get_real_badge_type(badge_type):
     return c.ATTENDEE_BADGE if badge_type in [c.PSEUDO_DEALER_BADGE, c.PSEUDO_GROUP_BADGE] else badge_type
 
 
+def add_opt(opts, other):
+    """
+    Add an option to a comma-separated string.
+    """
+    other = [other] if isinstance(other, str) else other
```

Comment:
Haha, I literally copied this from another place in the code! I'll, uh, update both of those places to use listify(). :) The suggested code looks great!